examples/plot_decoding_tutorial.py

```diff
@@ -78,41 +97,76 @@
+# 	whole brain signal is changing over time.
+#
+# We'll display the first three voxels by sub-selecting values from the 
+# matrix. You can also find more information on how to slice arrays `here
+# <https://docs.scipy.org/doc/numpy-1.13.0/reference/arrays.indexing.html#basic-slicing-and-indexing>`_.
+import matplotlib.pyplot as plt
+plt.plot(fmri_masked[5:150, :3])
+
+plt.title('Voxel Time Series')
+plt.xlabel('Scan number')
+plt.ylabel('Normalized signal')
+plt.tight_layout()
+
 ###########################################################################
 # Load the behavioral labels
-# ..........................
+# ...........................
 #
-# The behavioral labels are stored in a CSV file, separated by spaces.
+# Now that the brain images are converted to a data matrix, we can apply 
+# machine-learning to them, for instance to predict the task that the subject 
+# was doing. The behavioral labels are stored in a CSV file, separated by
+# spaces.
 #
 # We use pandas to load them in an array.
 import pandas as pd
 # Load behavioral information
-behavioral = pd.read_csv(haxby_dataset.session_target[0], sep=" ")
+behavioral = pd.read_csv(haxby_dataset.session_target[0], delimiter=' ')
 print(behavioral)
 
 ###########################################################################
-# Retrieve the experimental conditions, that we are going to use as
-# prediction targets in the decoding
+# The task was a visual-recognition task, and the labels denote the 
+# experimental condition: the type of object that was presented to the 
+# subject. This is what we are going to try to predict
 conditions = behavioral['labels']
-print(conditions)
+conditions
 
 ###########################################################################
 # Restrict the analysis to cats and faces
 # ........................................
 #
 # As we can see from the targets above, the experiment contains many
-# conditions, not all that interest us for decoding.
-#
-# To keep only data corresponding to faces or cats, we create a
-# mask of the samples belonging to the condition.
-condition_mask = conditions.isin(['face', 'cat'])
-
-# We apply this mask in the sampe direction to restrict the
-# classification to the face vs cat discrimination
-fmri_masked = fmri_masked[condition_mask]
+# conditions. As a consequence the data is quite big:
+print(fmri_masked.shape)
 
 ###########################################################################
-# We now have less samples
+# Not all of this data has an interest to us for decoding, so we will keep
+# only fmri signals corresponding to faces or cats. We create a mask of
+# the samples belonging to the condition, this mask is then applied to the
+# fmri data to restrict the classification to the face vs cat discrimination.
+# As a consequence, the input data is much less bigger (i.e. fmri signal is shorter):
```

Comment:
```suggestion
# As a consequence, the input data is much smaller:
```

Reply:
I thought it usefull to add a comment that the fmri signal is shorter to ensure that the reader is not loosing focus.
What do you think ?

Reply:
I think using "shorter" confuses me a bit in that we're not truncating the fMRI time series, but sub-selecting it. 

Can you think of another word, here ?